src/modules/dashboard/components/ExpenditurePage/Stages/LinkedMotions/LinkedMotions.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import Link from '~core/Link';
+
+import Tag from '~core/Tag';
+import styles from './LinkedMotions.css';
+
+const MSG = defineMessages({
+  linkedMotions: {
+    id: 'dashboard.Expenditures.Stages.linkedMotions',
+    defaultMessage: 'Linked motions',
+  },
+  foundExp: {
+    id: 'dashboard.Expenditures.Stages.foundExp',
+    defaultMessage: '{motion} Exp - {id}',
+  },
+  passed: {
+    id: 'dashboard.Expenditures.Stages.passed',
+    defaultMessage: 'Passed',
+  },
+  failed: {
+    id: 'dashboard.Expenditures.Stages.failed',
+    defaultMessage: 'Failed',
+  },
+  motion: {
+    id: 'dashboard.Expenditures.Stages.motion',
+    defaultMessage: 'Motion',
+  },
+});
+
+const displayName = 'dashboard.ExpenditurePage.Stages.LinkedMotions';
+
+interface Props {
+  status: 'passed' | 'failed' | 'pending';
```

Comment:
It should be an enum here

Reply:
You already have it declared, you just need to use it here instead of the strings